scheduler/src/manager.rs

```diff
@@ -0,0 +1,101 @@
+use log::{debug, info};
+use proto::scheduler::{
+    instance_service_server::InstanceServiceServer, node_service_server::NodeServiceServer,
+    Instance,
+};
+use tokio::task::JoinHandle;
+use tonic::transport::Server;
+
+use crate::{
+    instance_listener::InstanceListener, node_listener::NodeListener, storage::Storage, Node,
+};
+
+#[derive(Debug)]
+pub struct Manager {
+    instances: Storage<Instance>,
+    nodes: Storage<Node>,
+}
+
+impl Manager {
+    /// `new` creates a new `Manager` struct with two empty `Storage` structs
+    ///
+    /// Returns:
+    ///
+    /// A new Manager struct
+    pub fn new() -> Self {
+        Manager {
+            instances: Storage::new(),
+            nodes: Storage::new(),
+        }
+    }
+
+    /// This function returns a reference to the instances storage.
+    ///
+    /// Returns:
+    ///
+    /// A reference to the instances storage.
+    pub fn get_instances_storage(&self) -> &Storage<Instance> {
```

Comment:
Rust getters should simply be called instances()

Reply:
Fixed